exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/ApiControllerIntegrationTest.java

```diff
@@ -537,10 +510,7 @@ private Throwable logSafeExceptionMock(Class<? extends Throwable> exceptionType)
       VertxTestContext context) {
     return context.succeeding(
         response -> context.verify(() -> {
-          assertAll(
-              () -> assertThat(response.statusCode()).isEqualTo(HTTP_BAD_REQUEST),
-              () -> assertThat(response.bodyAsString()).startsWith("Transaction is not valid")
-          );
+          assertThat(response.statusCode()).isEqualTo(HTTP_INTERNAL_ERROR);
```

Comment:
I do not understand such changes in this method implementation — please clarify.

Reply:
Because it was a logic based on `InvalidTransactionException`. Such us we do not validate the transaction such error can't occur anymore.